br/pkg/storage/s3.go

```diff
@@ -276,7 +276,8 @@ func newS3Storage(backend *backuppb.S3, opts *ExternalStorageOptions) (obj *S3St
 	qs := *backend
 	awsConfig := aws.NewConfig().
 		WithS3ForcePathStyle(qs.ForcePathStyle).
-		WithRegion(qs.Region)
+		WithRegion(qs.Region).
+		WithS3Disable100Continue(true)
```

Comment:
will we merge this code to master?

Reply:
No. If we want to merge this to master, I think we need to add an argument `S3Disable100Continue` in `qs` and auto-adjust this argument through `s3.point` or `s3.provider`. This PR is used only for verification.

Reply:
but the hotfix code should merge into master according to release team

Reply:
Never heared of that before.... If we can't make sure this is the cause of this problem, how can we directly merge this into master?